CHANGELOG.md

```diff
@@ -30,7 +30,7 @@
 - [Pools] Disable manage button while wallet locked [#1877](https://github.com/thorchain/asgardex-electron/pull/1877)
 - Fix 24h volume [#1883](https://github.com/thorchain/asgardex-electron/pull/1883)
 - Fix outdated links [#1884](https://github.com/thorchain/asgardex-electron/pull/1884)
-- [Ledger] walletIndex get lost while veryfiying address on device [#1908](https://github.com/thorchain/asgardex-electron/issues/1908)
+- `walletIndex` gets lost [#1901](https://github.com/thorchain/asgardex-electron/pull/1901), [#1908](https://github.com/thorchain/asgardex-electron/issues/1908)
```

Comment:
Don't forget to add another line for the latest fix #1913

Reply:
Sorry, I've missed your comment. If current build is working well, let's add it to CHANGELOG afterwards in `develop` to avoid another CI run just to re-build everything just for CHANGELOG.

Reply:
:eye:  ea1208fdf9cabcee90873f735883274730869354